README.md

```diff
@@ -1,3 +1,20 @@
 # glibc-bootstrap
 
-TODO
+Bootstrap binaries for compiling `glibc` from source.
+
+## Requirements
+
+Linux with `glibc` 2.13 or newer.
```

Comment:
from what you've said: newer will not work as expected, right?

Reply:
I realize now that the way I've written this is confusing.  The requirements I wrote out were for what is needed to run the binaries as we've currently built them.  

Just to clarify:
- Someone could fork this repo and switch the Docker image to one with a newer `glibc`.  Everything would build fine but then the minimum `glibc` requirement for the binaries would be whatever the `glibc` version was of that alternative Docker image.  Our goal was to get the `glibc` version requirement below that of any non-EOL version of `glibc` but theoretically someone could use an Arch Linux image with the latest `glibc` and this would still work, though the binaries wouldn't be of much use for bootstrapping `glibc` on older systems.
- The `glibc` version of the host OS that is running Docker itself doesn't matter at all.  Within the container, the `glibc` that will be seen is the one that comes with Docker image, not the one of the the host OS that is running Docker.  Anyone with the ability to run `docker` on their system (or even an alternative Docker runtime like `singularity`) can build the Docker image and use it to run the build scripts.  

I'm guessing you were more interested in the second point.  I think we may need to split out the requirements into "what is needed to run the binaries this builds" and "what is needed to build the binaries" as these are completely different.

Reply:
> I think we may need to split out the requirements into "what is needed to run the binaries this builds" and "what is needed to build the binaries" as these are completely different.

👍🏻 good idea.

Reply:
I've added this now, along with a tutorial on how to build the binaries with Docker locally.